### ISSUE TITLE
Behat tests locally as well as on Travis-CI

### ISSUE THREAD

**Issue by arttus** [opendevshop/devshop#172]
It would be good to add some Behat tests for DevMaster and while at it, might as well add these to Travis-CI. #featurerequest

**Comment by arttus** [comment]
To be clear, there is this starting point https://github.com/opendevshop/devshop/tree/1.x/tests

**Comment by jonpugh** [comment]
You can run those tests locally now, and they are run in travis...

I even created a console command to kick off the testing job, so that we can automate generation of a behat profile to match the path and drush alias of any hostmaster install...

**Comment by jonpugh** [comment]
But you are right in that the tests need work, and need to be easier to run locally.

After this massive PR I'm working on is done I want to add a Robofile.php to the project instead of `development-launch.sh` scripts. "test" should be one of our dev console commands...

See http://robo.li/ it's amazing and we should be using it for everything.

**Comment by arttus** [comment]
What if we added a testing module, that can be enabled in Devshop? I was looking at the Behat tests in the Panopoly distro http://cgit.drupalcode.org/panopoly_test/tree/ as a example of this.

**Comment by jonpugh** [comment]
I hope we don't need a module just yet, until we need some special setup.  

I do like the idea of having a separate repo just for tests. This is what I did for aegir at https://github.com/aegir-project/tests

I'm focusing on improving tests today so I can make a release happen.

**Comment by jonpugh** [comment]
@arttus I've improved the local development process in the branch `robofile`, soon to be merged.

I've also altered the tests to not check for server hostname so they aren't hard coded for travis CI.

The robo CLI allows for super simple commands in a single file.

0. Install docker, docker-compose, and drush.
1. Install robo CLI: http://robo.li/
2. Clone this repo, checkout branch robofile: `git clone git@github.com:opendevshop/devshop`
3. cd into the folder and run `robo up`.
4. Run `robo test` to run the behat test suite.
